src/hotspot/os/windows/os_windows.hpp

```diff
@@ -188,18 +188,21 @@ class PlatformEvent : public CHeapObj<mtSynchronizer> {
 
 
 
-class PlatformParker : public CHeapObj<mtSynchronizer> {
-  protected:
-    HANDLE _ParkEvent ;
+class PlatformParker {
+  NONCOPYABLE(PlatformParker);
 
-  public:
-    ~PlatformParker () { guarantee (0, "invariant") ; }
-    PlatformParker  () {
-      _ParkEvent = CreateEvent (NULL, true, false, NULL) ;
-      guarantee (_ParkEvent != NULL, "invariant") ;
-    }
+ protected:
+  HANDLE _ParkerEvent;
```

Comment:
How about calling it _ParkHandle like PlatformEvent above it?  Two letters for me is not quickly distinguishable.  If you make this change I don't need to see another version.